src/VecSim/vec_sim_common.h

```diff
@@ -17,21 +17,37 @@ extern "C" {
 #include <stdbool.h>
 
 // Common definitions
-#define INVALID_ID    UINT_MAX
-#define INVALID_LABEL SIZE_MAX
-#define UNUSED(x)     (void)(x)
+#define DEFAULT_BLOCK_SIZE 1024
+#define INVALID_ID         UINT_MAX
+#define INVALID_LABEL      SIZE_MAX
+#define UNUSED(x)          (void)(x)
 
 // HNSW default parameters
 #define HNSW_DEFAULT_M       16
 #define HNSW_DEFAULT_EF_C    200
 #define HNSW_DEFAULT_EF_RT   10
 #define HNSW_DEFAULT_EPSILON 0.01
-#define DEFAULT_BLOCK_SIZE   1024
 
 #define HNSW_INVALID_LEVEL SIZE_MAX
 #define INVALID_JOB_ID     UINT_MAX
 #define INVALID_INFO       UINT_MAX
 
+// SVS-Vamana default parameters
+#define SVS_VAMANA_DEFAULT_ALPHA_L2                 1.2f
+#define SVS_VAMANA_DEFAULT_ALPHA_IP                 0.95f
+#define SVS_VAMANA_DEFAULT_GRAPH_MAX_DEGREE         32
+#define SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE 200
+#define SVS_VAMANA_DEFAULT_MAX_CANDIDATE_POOL_SIZE  (3 * SVS_VAMANA_DEFAULT_CONSTRUCTION_WINDOW_SIZE)
+#define SVS_VAMANA_DEFAULT_PRUNE_TO                 (SVS_VAMANA_DEFAULT_GRAPH_MAX_DEGREE - 4)
```

Comment:
what if SVS_VAMANA_DEFAULT_GRAPH_MAX_DEGREE is defined by the user differently than the default, but SVS_VAMANA_DEFAULT_PRUNE_TO is not?

Reply:
same as for SVS_VAMANA_DEFAULT_MAX_CANDIDATE_POOL_SIZE...